modules/ngap_module/NgapApi.cc

```diff
@@ -147,40 +147,68 @@ namespace ngap {
         vector<string> tokens;
         BESUtil::tokenize(restified_path, tokens);
         if (tokens.empty()) {
-            throw BESSyntaxUserError(string("The specified path '") + restified_path +
-                                     "' does not conform to the NGAP request interface API.", __FILE__, __LINE__);
+            throw BESSyntaxUserError(string("The specified NGAP API path '") + restified_path +
```

Comment:
While debugging the problem with the "/" in the collection name I discovered that the error handling code and the messages generated where identical for a number of importantly different cases. The changes in this section remedy this.